controllers/cloudscale.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2019, VSHN AG, info@vshn.ch
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/vshn/stack-cloudscale/controllers/s3"
+)
+
+// SetupWithManager adds all Cloudscale controllers to the manager.
+func SetupWithManager(mgr ctrl.Manager) error {
+	controllers := []interface {
+		SetupWithManager(ctrl.Manager) error
+	}{
+		&s3.BucketClaimSchedulingController{},
+		&s3.BucketClaimDefaultingController{},
+		&s3.BucketClaimController{},
+		&s3.BucketInstanceController{},
```

Comment:
Nitpick: I'd consider naming this `BucketController`, `BucketResourceController`, or similar. I think there's a little confusion about the term "instance" having meaning in Crossplane, which it does not. We have both claims named instance (e.g. MySQLInstance) and managed resources named instance (e.g. CloudSQLInstance).